server/routes/creatingReports/incidentDetails.test.ts

```diff
@@ -256,6 +290,7 @@ describe('POST save and return to tasklist', () => {
       })
   })
 
+  // is saving without incident date correct? - . Dev returns validation error
   test('Submitting without incident date is allowed', () => {
     return request(app)
```

Comment:
This test written was not written as part of this ticket. But I notice it states a user should be able to save and continue without to inputting an incident date. This currently presents a validation error message

Reply:
not with `save-and-return` which is what this is?